src/api/vectordb/transactions/repo.rs

```diff
@@ -21,7 +30,7 @@ pub(crate) async fn create_transaction(
 
     let transaction_id = vec_store
         .vcs
-        .add_next_version("main")
+        .generate_hash("new_transaction", Version::from(0))
```

Comment:
keep the branch to "main" and the version to "the current version of main branch" + 1, the [VersionControl::get_branch_info](https://github.com/cosdata/cosdata/blob/main/src/models/versioning.rs#L330) method returns [BranchInfo](https://github.com/cosdata/cosdata/blob/main/src/models/versioning.rs#L148) which have the `current_version` field.

Reply:
@a-rustacean could you please re-review it?